modules/op-starting-pipelines.adoc

```diff
@@ -19,7 +19,18 @@ For Pipelines created using the *From Git* option, the *Start Pipeline* dialog b
 ====
 +
 .. If you have resources in your namespace, the *Git Resources* and the *Image Resources* fields are prepopulated with those resources. If required, use the drop-downs to select or create the required resources and customize the Pipeline Run instance.
-.. If required, modify the *Advanced Options* and add credentials to authenticate the specified Git server or Docker registry.
+. Optional: Modify the *Advanced Options* and add credentials to authenticate the specified Git server or Docker registry.
```

Comment:
s/and add credentials/to add credentials
Probably add a few words when this is required, I think this is required when the user uses private git repo. Let's get this confirmed by the SME.

Reply:
@karthikjeeyar - COuld you please address this question . Thanks

Reply:
We need to add the secrets/credentials from the Advanced options, for the pipeline to get access to use the GIT/IMAGE resources which are private reposistory/registry.

Reply:
I think the current sentence is conveying this meaning.